templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/ObFinishView.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <cos-modal show>
+    <b-container class="modal-container">
+      <b-row align-h="center">
+        <b-col
+          v-for="connection in threeConnections()"
+          :key="connection.id"
+          align-self="center"
+          class="ob-connection-box"
+        >
+          <p class="ob-connection">{{ connection.name }}</p>
+          <span style="font-size:2.5rem;">{{ connection.avatar }}</span>
+        </b-col>
+      </b-row>
+      <b-row align-h="center">
+        <h3>
+          Great job! Here are some of
+          <br />
+          your connections.
+        </h3>
```

Comment:
Generally, it's better to avoid using line breaks in responsive design. I suggest adding a `max-width` css property to the `<h3>` instead.